src/views/components/your-tickets/attendees.php

```diff
@@ -0,0 +1,47 @@
+<?php
+/**
+ * Generic: Success Order Page Attendee list (Your Tickets).
```

Comment:
Given that this is a component, and it should be used in different places I'd avoid the `your-tickets` part, and use something more generic, like attendees list or similar.